models/health.go

```diff
@@ -0,0 +1,26 @@
+package models
+
+type HealthItemName string
+
+const (
+	DatabaseHealthItemName      HealthItemName = "Database"
```

Comment:
A bit pedanctic, but could we have slugs instead of full strings with uppercase letters and spaces?

If you look into `models/errors.go` (line 82), you already have some semblance of that with the `Requirement` system (and `open_sanctions` is already a constant there.